tool/cli.ts

```diff
@@ -992,8 +996,13 @@ if (Mozilla && !Mozilla.dntEnabled()) {
             invalidateCache: true,
             selective_mode: [cmdLC, macros],
           });
-        } catch (error) {
-          if (error.name === "MacroInvocationError") {
+        } catch (error: any) {
```

Comment:
Guess we don't need to type the `error` itself:

```suggestion
        } catch (error) {
```

Reply:
Fixed in 76056719f.